aodndata/acorn/handler.py

```diff
@@ -109,7 +109,12 @@ def preprocess(self):
             creation_date_nc_s3 = get_creation_date(os.path.join(self.opendap_root, destination_s3))
 
             if creation_date_nc_pipeline < creation_date_nc_s3:
+                self.logger.warning("incoming file '{dest_path}': creation date is older than file already on storage".
```

Comment:
Can we add in the message something like "publish_type set to NO_ACTION"?

Reply:
That kind of message is already logged elsewhere in an INFO message, since there is a callback which prints every file status change automatically, for example:

```
2018-08-14 12:49:32,353 INFO [root] updated file 'IMOS_ACORN_V_20170101T003000Z_ROT_FV01_1-hour-avg.nc': {'publish_type': 'NO_ACTION'}
```

So the warning message is really to complement this standard state change message.

Reply:
Ok thanks, didn't see this in the logs.